public/docs/ts/latest/guide/security.jade

```diff
@@ -200,11 +200,12 @@ h2#http HTTP-level vulnerabilities
 
 h3#xsrf Cross-site request forgery
 :marked
-  In a cross-site request forgery, an attacker tricks the user into visiting a
-  _different_ page and has them, for example, submit a form that sends a request to your application's
-  web server. If the user is logged into your application, the browser will send authentication
-  cookies, and the attacker could&mdash;for example&mdash;cause a bank transfer in the user's name with
-  the right request.
+  In a cross-site request forgery (CSRF or XSRF), an attacker tricks the user into visiting
+  a different web page with malicious code that secretly sends a request to your application's web server. 
```

Comment:
this might be clearer if you give the websites names, e.g. `evil.com` and `example.com`.